stable/prometheus-redis-exporter/Chart.yaml

```diff
@@ -1,8 +1,8 @@
 apiVersion: v1
-appVersion: 0.28.0
+appVersion: 1.0.3
 description: Prometheus exporter for Redis metrics
 name: prometheus-redis-exporter
-version: 2.0.2
+version: 2.1.0
```

Comment:
We better bump major version here as this is a breaking change.

Reply:
true